pyVmomi/VmomiSupport.py

```diff
@@ -460,6 +460,8 @@ def __getattr__(self, name):
    def __eq__(self, other):
       if other is None:
          return False
+      elif type(other) is str:
```

Comment:
don't see the benefit here as it would still be a str object, in python2 and in python3. Do I miss something?